vector-stores/spring-ai-pinecone-store/src/main/java/org/springframework/ai/vectorstore/PineconeVectorStore.java

```diff
@@ -298,6 +303,8 @@ public static final class PineconeVectorStoreConfig {
 
 		private final String contentFieldName;
 
+		// TODO: Why is this field configurable? Can we remove this after standardizing
```

Comment:
Would it be ok to remove this and keep the standard "distance" metadata? Having this configurable means we cannot use the metadata reliably across implementations.